static/app/utils/profiling/renderers/textRenderer.tsx

```diff
@@ -126,15 +126,14 @@ class TextRenderer {
       // If the width of the text is greater than the minimum width to render, we should render it
       if (paddedRectangleWidth >= minWidth) {
         let text = frame.frame.name;
-        const textWidth = this.measureText(this.context, text);
 
         // If text width is smaller than rectangle, just draw the text
-        if (textWidth > paddedRectangleWidth) {
+        if (this.measureText(this.context, text) > paddedRectangleWidth) {
           text = trimTextCenter(
             text,
             findRangeBinarySearch(
-              {low: 0, high: text.length},
-              n => this.measureText(this.context, text.substring(0, n)),
+              {low: 0, high: paddedRectangleWidth},
+              n => this.measureText(this.context, text.substring(0, n + 1)),
```

Comment:
Is this `+ 1` to ensure the text will fit inside of the `paddedRectangleWidth` despite only rendering `n` characters?

Reply:
Actually, I can remove this.